js/main.js

```diff
@@ -153,3 +177,21 @@ document.addEventListener('DOMContentLoaded', function (event) {
     viewSwapDataViews(data.view);
   }
 });
+
+// eventlistener 'click' for document
+document.addEventListener('click', function (event) {
+  var links = document.getElementsByTagName('a');
+
+  for (var i = 0; i < links.length; i++) {
+    if (event.target.tagName !== links[i].tagName) {
```

Comment:
You know what the value of the `tagName` property would be for an `a` element, so this loop is not needed. You can simply compare against the value you want to check for: `"A"`